config/features.yaml

```diff
@@ -75,3 +75,14 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+# Linkedin account extraction from profile edit url
+linkedin_accounts:
+        query_string: 'source_short:"WEBHIST" AND
+          url:"https://www.linkedin.com/in/" AND url:"/edit/"'
+        attribute: 'url'
+        store_as: 'found_account'
+        re: 'https://www.linkedin.com/in/([A-z-\d]{5,32})/edit/'
+        aggregate: True
+        tags: ['LinkedIn Account']
```

Comment:
```suggestion
        tags: ['linkedin-account']
```

Reply:
we are in the process of standardize how we handle tags and will move to kebab-case. Let's change this one to "linkedin-account"